build-parent/pom.xml

```diff
@@ -129,6 +209,13 @@
     </plugins>
   </build>
 
+  <dependencies>
```

Comment:
Since I am bringing `build-resources` under the `root` pom, I had to move this dependency to the child parent poms.